examples/cpp_api/quickstart_dense.cc

```diff
@@ -114,8 +114,10 @@ int main() {
   if (Object::object(ctx, array_name).type() != Object::Type::Array) {
     create_array();
     write_array();
+  } else {
+    std::cout << "Skipping existing array (delete to re-write)" << std::endl;
```

Comment:
I would remove these changes, just to keep the examples consistent with one another.